docs/elements/guides/sign-in.mdx

```diff
@@ -0,0 +1,290 @@
+---
+description: Learn how to build a complete sign-in form with Clerk Elements.
+---
+
+# Build a sign-in flow with Clerk Elements
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Follow the Next.js quickstart",
+      link: "https://clerk.com/docs/quickstarts/nextjs",
+    },
+    {
+      title: "Upgrade to Core 2 (if necessary)",
+      link: "https://clerk.com/docs/upgrade-guides/core-2/nextjs",
+    },
+  ]}
+>
+
+- Build out different steps of an authentication flow
+- Accept user input through form fields
+- Verify a user's identity during a sign-in attempt
+- Support various password-related features
+
+</TutorialHero>
+
+<Callout type="info">
+
+- Clerk Elements is for [advanced use-cases](/docs/elements/overview#why-use-clerk-elements) that require a high-level of customization. The easiest way to implement Clerk is with our [all-in-one UI components](/docs/components/overview).
+
+- Clerk Elements currently only works with Next.js and [Clerk Core 2](https://clerk.com/changelog/2024-04-19). As it gets closer to a stable release, support for additional frameworks will be added.
+
+</Callout>
+
+<Steps>
+
+### Add a sign-in route
+
+Create a new route in your Next.js application. The route needs to be an [optional catch-all route](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments) so the sign-in flow can handled nested paths, as shown in the following example:
```

Comment:
We removed it because we shouldn't have repetition like that in prose - the codeblocks were recently redesigned so that the filename and copy button can stand out more so hopefully that should help users more with seeing the filename!